.github/ISSUE_TEMPLATE/new_idea_proposal.yml

```
@@ -0,0 +1,30 @@
+---
+name: New idea or Proposal 
+description: Suggest an idea or submit a new proposal to improvise the Ansible Community.
+
+- type: textarea
+  attributes:
+    label: Summary
+    description: Explain about the new idea or proposal breifly below.
+    placeholder: >-
+      Details about the new idea/proposal and links to GitHub issues/PR if any.
```

```suggestion
      Details about the new idea/proposal and links to GitHub issues/PRs if any.
```
I'd avoid abbreviations. Many people don't know what is the `PR`.